File: python/NpowerN.py
Number = int(input("Enter a number to multiply the number itself is the number of times:"))
Counter = 1
Multiple = 1
while(Counter <= Number):
  Multiple = Multiple * Number
  Counter = Counter + 1
print(str(Number) + " Multiplied " + str(Number) + " times is " + str(Multiple) + ".")

File: swift/NpowerN.swift
var Counter = 1;
var Multiple = 1;
print("Enter a number to multiply the number itself is the number of times:", terminator : "");
var Count = Int(readLine()!)! 
while(Counter <= Count) {
  Multiple = Multiple * Count;
  Counter = Counter + 1;
}
print("\(Count) multiplied \(Count) times is \(Multiple).");
 